repose-aggregator/commons/commons-configuration/src/test/resources/unmarshallerValidator/xsd/header-user-configuration.xsd

```diff
@@ -20,51 +20,47 @@
   -->
 
 
-<xs:schema xmlns="http://docs.openrepose.org/repose/header-id-mapping/v1.0"
-           xmlns:html="http://www.w3.org/1999/xhtml"
-           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+<xs:schema xmlns:header-user="http://docs.openrepose.org/repose/header-user/v1.0" xmlns:html="http://www.w3.org/1999/xhtml"
```

Comment:
If you make this the default namespace (drop the :header-user) you won't have to include the namespace when referring to your own types (e.g., header-user:HeaderUserConfig).